exporters/etw/src/etw_provider_exporter.cc

```diff
@@ -0,0 +1,3 @@
+#define HAVE_NO_TLD
```

Comment:
You can also pull these files into Bazel build, but have a check on very top and bottom as

```cpp
#ifdef _WIN32
...
#endif
```

Of course this is not as good as configuring it at Bazel-build system level, but then you'd have one configuration across all OS: you pull in the file at all times, but it compiles into an empty object.